lib/src/tools/display/display_service.dart

```diff
@@ -1,28 +1,42 @@
-abstract class DisplayService {
-  /// Gets whether Bluetooth adapter is turn ON.
-  int get brightness;
-
-  ///
-  /// Call this method to enable Bluetooth adapter
-  ///
-  void setBrightness(int value);
-}
+import "display_service_contract.dart" show DisplayService, DisplayDevice;
 
 class DisplayServiceStub extends DisplayService {
   @override
-  int brightness;
+  final List<DisplayDevice> displays;
+
+  @override
+  final bool isDarkModeEnabled;
 
-  DisplayServiceStub() : this.brightness = 50;
+  @override
+  final bool isNightModeEnabled;
 
   @override
-  void setBrightness(int value) {
-    if (value < 0 || value > 100) {
-      throw ArgumentError.value(value);
+  void setBrightness(DisplayDevice display, double brightness) {
+    if (brightness < 0 || brightness > 100) {
+      throw ArgumentError.value(brightness);
     }
+  }
 
-    this.brightness = value;
+  @override
+  bool isEnabled;
```

Comment:
Why do you declare property that never used?

Reply:
Removed property.